.github/workflows/sync_crowdin.yml

```diff
@@ -38,16 +40,15 @@ jobs:
       - name: Remove empty values from JSON
```

Comment:
@Ushie What is this step used for?

Reply:
Removing empty strings in the JSONs exported from Crowdin, because Crowdin doesn't have a double-nested JSON post-processor to remove them

Reply:
Can you show an example of how such a file looks like that needs this post processing?